src/test/java/uk/gov/hmcts/reform/sscs/helper/SscsHelperTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.YES;
+import static uk.gov.hmcts.reform.sscs.helper.SscsHelper.updateDirectionDueDateByAnAmountOfDays;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import junitparams.JUnitParamsRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+
+@RunWith(JUnitParamsRunner.class)
```

Comment:
This is not required (you have not used @Parameters). And besides should be avoided in favour of @ParameterizedTest.